### PR TITLE
Remove admission code

### PR DESCRIPTION
**Pull request by HomayoonAlimohammadi**
# Remove admission code
This PR removes any admission-related logic. This logic was introduced but never enabled in production.

We have decided to remove this feature, which means that the following flags are removed in future releases:
- admission-control-policy
- admission-control-policy-limit
- admission-control-only-for-write-queries

Anyone currently making use of this feature in k8s-dqlite `v1.2.0` will be expected to drop these flags before updating to a newer version of k8s-dqlite.

### CODE REVIEW

**Review by bschimke95 — APPROVED** [review]
LGTM
Just a note: This is a breaking change if anybody would use the admission code. I don't think so (as it was never used) but we definitely need to mention this in the next k8s-dqlite release.

**Review by louiseschmidtgen — APPROVED** [review]
Thanks Hue! Let's drop a note in the configuration doc and then we are good to go.

**Review by marco6 — APPROVED** [review]

File: docs/configuration.md [hunk 1]
```diff
@@ -28,9 +28,6 @@ The following configuration options are available listed in a table format:
 | `--watch-storage-available-size-interval` | Interval to check if the disk is running low on space | `5s` |
 | `--watch-storage-available-size-min-bytes` | Minimum required available disk size (in bytes) to continue operation | `10*1024*1024`|
 | `--low-available-storage-action` | Action to perform in case the available storage is low | `none` |
-| `--admission-control-policy` | Transaction admission control policy to use | `allow-all` |
-| `--admission-control-policy-limit` | Maximum number of transactions that are allowed to run concurrently | `300` |
-| `--admission-control-only-for-write-queries` | If set, admission control will only be applied to write queries | `false` |
```
[the remark this answers is not in the document]
Comment (HomayoonAlimohammadi) [reply docs/configuration.md:33]: I crossed out these flags and described them as `REMOVED`. WYDT?

[the remark this answers is not in the document]
Comment (louiseschmidtgen) [reply docs/configuration.md:33]: Perfect, thank you!

**Pull request merged**